### PR TITLE
Modifications to Estimator::CostForce + minor fixes

### PR DESCRIPTION
**Pull request by thowell**
Modifies `Estimator::CostForce` to consider position and rotational components of forces resulting from free joints. 
Minor fix to initialization of `norm_parameters_force_`.

### CODE REVIEW

File: grpc/estimator_service.cc [hunk 1]
```diff
@@ -12,10 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <cstring>
```
Comment (erez-tom) [inline grpc/estimator_service.cc:15]: This change to the header order will upset the internal linter.  Can you please undo it, and perhaps more deeply we should figure out how to alter the style spec on your side so this doesn't happen again...

**Review by erez-tom — CHANGES_REQUESTED** [review]
only the header order is blocking, the other comment is just a question.

File: mjpc/estimators/estimator.cc [hunk 2]
```diff
@@ -53,12 +53,24 @@ void Estimator::Initialize(mjModel* model) {
   configuration_prior_.Initialize(nq, configuration_length_);
 
   // sensor
-  dim_sensor_ = model->nsensordata;  // TODO(taylor): grab from model
-  num_sensor_ = model->nsensor;      // TODO(taylor): grab from model
+  dim_sensor_ = model->nsensordata;  // TODO(taylor): grab from xml
```
[the remark this answers is not in the document]
Comment (thowell) [reply mjpc/estimators/estimator.cc:56]: My thinking is that if we want to have a task xml that specifies both a planner and estimator, we will need to figure out which sensors are actually sensors and not custom costs for residual terms. 

If we always use a separate xml for the estimator then we can directly grab the correct dimensions for the model.

File: grpc/estimator_service.cc (hunk 1 above)
Comment (erez-tom) [reply grpc/estimator_service.cc:15]: Resolving so I can merge, but I'll have to manually undo this internally, so we should find a way to make sure it doesn't happen routinely.

**Review by erez-tom — APPROVED** [review]

**Pull request merged**